packages/apps/shopify-app-remix/src/server/authenticate/admin/types.ts

```diff
@@ -102,6 +103,11 @@ interface AdminContextInternal<
    * ```
    */
   cors: EnsureCORSFunction;
+
+  /**
+   * Methods to manage optional scopes for the store that made the request.
+   */
+  scopes: ScopesApiContext;
```

Comment:
Once we nail down the types / API for this, we should also make sure to add an example for the docs.

Reply:
Totally agree. However I'd wait until we have the complete api defined, what do you think?